src/softfloat.h

```diff
@@ -24,7 +24,14 @@ enum {
     FFLAG_UNDERFLOW   = 0b00000000000000000000000000000010,
     FFLAG_INEXACT     = 0b00000000000000000000000000000001,
     //                    ....xxxx....xxxx....xxxx....xxxx
-    RV_NAN            = 0b01111111110000000000000000000000
+    RV_NAN            = 0b01111111110000000000000000000000,
+    //                    ....xxxx....xxxx....xxxx....xxxx
```

Comment:
I am not sure if the following identifiers make sense. Can you explicitly use them somewhere?

Reply:
Maybe I can use switch case to compare them directly when I need to.
Should I delete this?

Reply:
You can implement the decoding for rounding mode in the proposed changes.

Reply:
Any progress?